Termux Android cannot find "golang.org/x/sys/unix"

On Termux Android, the lib depends to `go-issaty` is failing. It cannot find `golang.org/x/sys/unix`. I might relate to https://github.com/mattn/go-isatty/pull/28?

```
go: finding github.com/stretchr/testify/mock latest
../../../go/pkg/mod/github.com/mattn/go-isatty@v0.0.5/isatty_linux.go:6:8: unknown import path "golang.org/x/sys/unix": cannot find package
```

I think it may be related to the compilation constraints _somewhere_ (either in this library or in `golang.org/x/sys`).

I don't believe it's related to #28. That PR only made the problem more explicit, the code wouldn't compile with `v0.0.4` of this library either.

**`main.go`**
```go
package main

import (
	"fmt"
	"os"

	"github.com/mattn/go-isatty"
)

func main() {
	fmt.Printf("%t", isatty.IsTerminal(os.Stdout.Fd()))
}
```

```sh
$ go mod init
go: creating new go.mod: module github.com/radeksimko/test

$ go get ./...
go: finding github.com/mattn/go-isatty v0.0.5
go: downloading github.com/mattn/go-isatty v0.0.5
go: finding golang.org/x/sys v0.0.0-20190222072716-a9d3bda3a223
go: downloading golang.org/x/sys v0.0.0-20190222072716-a9d3bda3a223

$ go mod tidy

$ GOOS=android GOARCH=arm64 go build .
/go/pkg/mod/github.com/mattn/go-isatty@v0.0.5/isatty_linux.go:6:8: unknown import path "golang.org/x/sys/unix": cannot find package
```
_downgrading_
```sh
$ go get github.com/mattn/go-isatty@v0.0.4
go: finding github.com/mattn/go-isatty v0.0.4
go: downloading github.com/mattn/go-isatty v0.0.4

$ GOOS=android GOARCH=arm64 go build .
# github.com/radeksimko/test
/usr/local/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/usr/bin/ld: /tmp/go-link-981758022/go.o: Relocations in generic ELF (EM: 183)
/tmp/go-link-981758022/go.o: error adding symbols: File in wrong format
collect2: error: ld returned 1 exit status
```

--- a/isatty_android.go
+++ b/isatty_android.go
@@ -1,0 +1,17 @@
+// +build android
+
+package isatty
+
+import (
+	"syscall"
+	"unsafe"
+)
+
+const ioctlReadTermios = syscall.TCGETS
+
+// IsTerminal return true if the file descriptor is terminal.
+func IsTerminal(fd uintptr) bool {
+	var termios syscall.Termios
+	_, _, err := syscall.Syscall6(syscall.SYS_IOCTL, fd, ioctlReadTermios, uintptr(unsafe.Pointer(&termios)), 0, 0, 0)
+	return err == 0
+}

--- a/isatty_linux.go
+++ b/isatty_linux.go
@@ -1,5 +1,6 @@
 // +build linux
 // +build !appengine
+// +build !android
 
 package isatty
 

